routers/repo/pull.go

```diff
@@ -540,6 +540,9 @@ func CompareAndPullRequest(ctx *middleware.Context) {
 	ctx.Data["PageIsComparePull"] = true
 	ctx.Data["IsDiffCompare"] = true
 	ctx.Data["RequireHighlightJS"] = true
+	setTemplateIfExists(ctx, ".github/PULL_REQUEST_TEMPLATE.md", "PullRequestTemplate")
```

Comment:
Why would you call 3 times in the handler? Shouldn't we make a function that make 3 attempts and stop once found?